### ISSUE TITLE
 Capacitor WebView loaded ⚡️  [warn] - Ionic Native: deviceready did not fire within 5000ms.

### ISSUE THREAD

**Issue by diadal** [quasarframework/quasar#9070]
Both on iOs and Android white screen not addition logs  
WebView loaded
⚡️  [warn] - Ionic Native: deviceready did not fire within 5000ms. This can happen when plugins are in an inconsistent state. Try removing plugins from plugins/ and reinstalling them.

`"quasar": "^2.0.0-beta.13",` `"@quasar/app": "^3.0.0-beta.17",`
`
"@capacitor/app": "^0.3.3",
    "@capacitor/cli": "^3.0.0-beta.1",
    "@capacitor/core": "^3.0.0-beta.1",
    "@capacitor/device": "^0.5.6",
    "@capacitor/local-notifications": "^0.6.0",
    "@capacitor/push-notifications": "^0.3.6",
    "@capacitor/splash-screen": "^0.3.3",
    "@capacitor/status-bar": "^0.4.6",
`

**Comment by diadal** [comment]
have remove all the Ionic plugins now this 
<img width="496" alt="Screen Shot 2021-04-28 at 8 48 00 AM" src="https://user-images.githubusercontent.com/19895516/116366790-9055b580-a7fe-11eb-8f71-7e6d77426806.png">
<img width="438" alt="Screen Shot 2021-04-28 at 8 49 09 AM" src="https://user-images.githubusercontent.com/19895516/116366840-a06d9500-a7fe-11eb-98ff-96c609e4c49a.png">

**Comment by diadal** [comment]
this fix my issue `import { defineComponent, onMounted, h } from 'vue';`

```
/**
 * THIS FILE IS GENERATED AUTOMATICALLY.
 * DO NOT EDIT.
 *
 * You are probably looking on adding startup/initialization code.
 * Use "quasar new boot <name>" and add it there.
 * One boot file per concern. Then reference the file(s) in quasar.conf.js > boot:
 * boot: ['file', ...] // do not add ".js" extension to it.
 *
 * Boot files are your "main.js"
 **/

import { Quasar } from 'quasar';
import quasarUserOptions from './quasar-user-options.js';
import AppComponent from 'app/src/App.vue';

import createStore from 'app/src/store/index';

import createRouter from 'app/src/router/index';

import '@capacitor/core';

// importing it so it can install itself (used by Quasar UI)
import { App as CapApp } from '@capacitor/app';

// import { SplashScreen } from '@capacitor/splash-screen';

import { defineComponent, onMounted, h } from 'vue';
const RootComponent = defineComponent({
  name: 'AppWrapper',
  setup(props) {
    onMounted(() => {
      SplashScreen.hide();
    });

    return () => h(AppComponent, props);
  }
});

export default async function(createAppFn) {
  // create store and router instances

  const store =
    typeof createStore === 'function' ? await createStore({}) : createStore;

  // obtain Vuex injection key in case we use TypeScript
  const { storeKey } = await import('app/src/store/index');

  const router =
    typeof createRouter === 'function'
      ? await createRouter({ store })
      : createRouter;

  // make router instance available in store
  store.$router = router;

  // Create the app instance.
  // Here we inject into it the Quasar UI, the router & possibly the store.
  const app = createAppFn(RootComponent);

  app.config.devtools = true;

  app.use(Quasar, quasarUserOptions);

  app.config.globalProperties.$q.capacitor = window.Capacitor;

  // Expose the app, the router and the store.
  // Note that we are not mounting the app here, since bootstrapping will be
  // different depending on whether we are in a browser or on the server.
  return {
    app,
    store,
    storeKey,
    router
  };
}

```

please 1 question how can i disable this or remove this 

```
 onMounted(() => {
      SplashScreen.hide();
    });
```

**Comment by rstoenescu** [comment]
`onMounted` is already imported. maybe you're not running with the latest q/app version?

**Comment by rstoenescu** [comment]
Fix will be available in "@quasar/app" v3.0.0-beta.18
